src/assets/icons/taskbar/search-dark.svg

```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="-2 -1 26 26">
+    <g fill="#ffffff30" stroke="none">
+        <circle cx="13" cy="11" r="8"></circle>
+    </g>
+    <g fill="none" stroke="#ffffff" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
```

Comment:
We can actually have a single search icon image by using `stroke="currentColor"`.

Reply:
It will change the color of the icon based on the color provided by the css.

Reply:
well no, you used svg to be img src url so we can't style it!!!!!!!!!!!!!!!!!!

Reply:
Ohh, my bad.